src/package.json

```diff
@@ -9,6 +9,10 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "@umbraco-cms/backoffice": "^14.0.0--preview007",
```

Comment:
This is still referencing a preview version.

Reply:
I guess it and the package-lock.json next to it isn't needed right?  As the one you are using is in "Client".